### ISSUE TITLE
Tox installs dependencies from setup.py

### ISSUE THREAD

**Issue by rupert** [renalreg/radar#198]
Should be using `requirements.txt` instead.

Use something like `pip install --no-deps`.

**Comment by rupert** [comment]
```
GLOB sdist-make: /home/vagrant/src/api/setup.py
pytest create: /home/vagrant/src/api/.tox/pytest
pytest develop-inst: /home/vagrant/src/api
pytest installed: Flask==0.10.1,itsdangerous==0.24,Jinja2==2.8,MarkupSafe==0.23,radar==0.3,radar-api==0.5.3,six==1.10.0,SQLAlchemy==1.0.9,Werkzeug==0.11.2,wheel==0.24.0
pytest runtests: PYTHONHASHSEED='793686929'
pytest runtests: commands[0] | pip install --no-deps --editable ../radar
```

**Comment by rupert** [comment]
Fixed in 8ad6a9bb1e78ef85eaecfdf9f0e320b37e595eb7
